README.md

```diff
@@ -144,17 +150,19 @@ It is time to setup the nodes of the cluster.
 
 Take a declarative approach and use [`terraform`](https://www.terraform.io) to setup the nodes of the cluster.
 Note that in this example I utilize the AWS Spots; you can easily change the code and use the regular on-demand instances.
-This is left as an exercise.
+
+This is left as an exercise. <!-- maybe give a hint here towards the solution or let people know that the solution is in the GitHub repo --->
+
 I use two groups of file to define the cluster:
 
 - `.tf` instructions: parsed by `terraform` and defining what instances to use, what tags, regions, etc.
 - Provisioning shell scripts: installing needed tools on the nodes
 
 ### `.tf` files
 
-When using `terraform` all `.tf` files are read and concatenated.
-There are more details of course; a good entry point would be [here](https://www.terraform.io/docs/configuration/index.html).
-In our example I organized the `.tf` files as follows:
+When using `terraform` all `.tf` files are read and concatenated.There are more details of course; a good entry point would be [here](https://www.terraform.io/docs/configuration/index.html).
```

Comment:
Again, missing a '.'

Reply:
Not sure what you mean by this?

Reply:
@Kacawi I meant a missing space between "concatenated." and "There". But it seems like you fixed it.